build/azure-pipelines/upload-sourcemaps.js

```diff
@@ -13,19 +13,39 @@ const util = require('../lib/util');
 const root = path.dirname(path.dirname(__dirname));
 const commit = util.getVersion(root);
 
-function main() {
-	const vs = vfs.src('out-vscode-min/**/*.map', { base: 'out-vscode-min' }) // client source-maps only
+// optionally allow to pass in explicit base/maps to upload
+const args = process.argv.slice(2);
+const base = args[0];
+const maps = args[1];
```

Comment:
Where are your ES3000 skillz? :1st_place_medal: 

```ts
const [base, maps, ...args] = process.argv;
```

Reply:
@joaomoreno I guess in the same place where your node.js skillz are.

Went with this:
```js
const [base, maps] = process.argv.slice(2);
```

Reply:
Ahah yeah `const [,,base, maps] = process.argv;`